includes/multi-currency/MultiCurrency.php

```diff
@@ -1441,28 +1441,56 @@ public function is_multi_currency_settings_page(): bool {
 		);
 	}
 
+	/**
+	 * Function used to compute the customer used currencies, used as internal callable for get_all_customer_currencies function.
+	 *
+	 * @return array
+	 */
+	public function callable_get_customer_currencies() {
```

Comment:
Since this is only meant to be called within this class, maybe we should update it to be a private method.

Reply:
Getting this error when private method is used

```
1) WCPay_Multi_Currency_Tests::test_get_all_customer_currencies
TypeError: Argument 2 passed to Mock_Database_Cache_342a5a7d::get_or_add() must be callable, array given, called in /var/www/html/wp-content/plugins/woocommerce-payments/includes/multi-currency/MultiCurrency.php on line 1499
```

Reply:
Okay, I see where the issue is. I didn't initially realize that the method name was being passed outside of the class itself to be fired off elsewhere. The only way we would be able to resolve the error would be to duplicate code.